promitor-agent-scraper/values.yaml

```diff
@@ -94,12 +94,24 @@ health:
     enabled: true
     verifyDependencies: false
     delay: 5
+    failureThreshold: 3
     interval: 5
+    successThreshold: 1
+    timeoutSeconds: 1
```

Comment:
How about this?
```suggestion
    interval: 5
    timeoutSeconds: 1
    thresholds:
      success: 1
      failure: 3
```

Reply:
If we change this, let's do it for all probes

Reply:
_I know this is the default for probes, but it's not exactly inline with the rest of the values_

Reply:
Don't have any objections, changed.